src/plugin/NextSitemapWebpackPlugin.js

```diff
@@ -6,17 +6,14 @@ import { ConcatSource } from 'webpack-sources';
 class NextSitemapWebpackPlugin {
     apply(compiler) {
         compiler.hooks.emit.tapPromise('NextSitemapPlugin', async (compilation) => {
-            const diskRoutes = glob.sync('pages/**/*.js', { ignore: ['pages/api/**', 'pages/_*.js'] });
+            const diskRoutes = glob.sync('pages/**/*.(js|jsx|mjs|ts|tsx)', { ignore: ['pages/api/**', 'pages/_*'] });
```

Comment:
A `+` is required before the parenthesis to indicate that you want one or more occurrence within the patterns given, otherwise glob will look for a literal `(js|jsx|mjs|ts|tsx)` extension that won't exist and this will cause things to break

Reply:
Fixed.